lib/fluent/test/helpers.rb

```diff
@@ -45,6 +45,14 @@ def msgpack(type)
           raise ArgumentError, "unknown msgpack object type '#{type}'"
         end
       end
+
+      def str2time(str_time, format = nil)
```

Comment:
Could you add keyword argument `format` to `event_time` instead of adding another method `str2time`?